simpleocr/files.py

```diff
@@ -45,45 +32,152 @@ def write(self):
         write_boxfile(self.path, self.classes, self.segments)
 
 
-class ImageFile(object):
+class Image(object):
     """
-    An OCR image file. Has an image and its file path, and optionally
-    a ground (ground segments and classes) and it's file path
+    Complete class that contains functions for creation from file,
+    as well as from PIL Images. Also supports grounding in memory.
     """
-
-    def __init__(self, image_path):
-        if not os.path.exists(image_path):
-            raise IOError("Image file not found: " + image_path)
-        self.image_path = image_path
-        self.image = cv2.imread(self.image_path)
-        basepath = os.path.splitext(image_path)[0]
-        self.ground_path = try_extensions(GROUND_EXTENSIONS, basepath)
-        if self.ground_path:
-            self.ground = GroundFile(self.ground_path)
-            self.ground.read()
+    def __init__(self, array=None, path=None, debug=False):
```

Comment:
Separating into two classes and migrating the fuzzy-file-loading should make the size of `__init__` much more manageable. Right now it's too large